iothub/service/src/Http/HttpClientHelper.cs

```diff
@@ -885,5 +886,13 @@ internal static HttpMessageHandler CreateDefaultHttpMessageHandler(IWebProxy web
 
             return httpMessageHandler;
         }
+
+        private static string GetClientVersion()
```

Comment:
I don't see the point of decentralizing this when we now have 2 copies of the same code.